homeassistant/components/bosch_shc/config_flow.py

```diff
@@ -205,6 +210,8 @@ async def async_step_zeroconf(self, discovery_info):
         local_name = discovery_info[zeroconf.ATTR_HOSTNAME][:-1]
         node_name = local_name[: -len(".local")]
 
+        if TYPE_CHECKING:
+            assert self.info
```

Comment:
I think it would make more sense to change the check at line 205, to check both info & host against None.

Reply:
@frenck: done